python/tests/test_network.py

```diff
@@ -3334,7 +3344,7 @@ def test_variable_limits(self):
             self.assertEqual(x.size,net.num_vars)
             for t in range(self.T):
                 for bus in net.buses:
-                    self.assertEqual(x[bus.index_v_mag[t]],bus.v_min)
+                    self.assertEqual(x[bus.index_v_mag[t]],bus.v_min_reg)
```

Comment:
Here in test_variable_limits, I do not think it is correct that you are using v_max/min_reg. Please check. These limits should also be for regulation-related functions or constraints.